src/actions/security-chat.js

```diff
@@ -59,19 +87,67 @@ export function chatMessageIsTyping(boolean) {
     type: CHAT_MESSAGE_IS_TYPING,
   };
 }
-// export function chatRemoveMessage(messageId) {
-//   return {
-//     messageId: action.messageId,
-//     type: CHAT_REMOVE_MESSAGE,
-//   };
-// }
+export function chatRemoveMessage(messageId) {
+  return {
+    messageId,
+    type: CHAT_REMOVE_MESSAGE,
+  };
+}
+export function setUpdatedMessage(messageId, updatedMessage) {
+  return {
+    messageId,
+    message: updatedMessage,
+    isTyping: false,
+    type: CHAT_UPDATE_MESSAGE_BY_ID,
+  };
+}
+export function removeMessage(messageId) {
+  chatRemoveMessage(messageId);
+  return async () => {
+    try {
+      const response = await fetch(
+        `http://localhost:8080/api/message/${messageId}`,
+        {
+          method: 'delete',
+        },
+      );
+      const result = await response.json(response);
+      console.info('Delete result', result);
+      return result;
+    } catch (error) {
+      return error;
+    }
+  };
+}
+export function updateMessage(messageId, updatedMessage) {
+  return async dispatch => {
+    dispatch(fetchDataStart());
+    try {
+      const response = await fetch(
+        `http://localhost:8080/api/message/${messageId}`,
+        {
+          method: 'put',
+          body: JSON.stringify(updatedMessage),
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        },
+      );
+      const result = await response.json(response);
```

Comment:
Please read the docs and remove this redundant argument everywhere https://developer.mozilla.org/ru/docs/Web/API/Fetch_API/Using_Fetch